src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Atom10FeedFormatter.cs

```diff
@@ -1154,15 +1152,15 @@ private void ReadPersonFrom(XmlReader reader, SyndicationPerson result)
                     {
                         if (reader.IsStartElement(Atom10Constants.NameTag, Atom10Constants.Atom10Namespace))
                         {
-                            result.Name = reader.ReadElementString();
+                            result.Name = StringParser(reader.ReadElementString(), Atom10Constants.NameTag, reader.NamespaceURI);
```

Comment:
reader.NamespaceURI will do an extra property fetch which is unnecessary as you know it will be Atom10Constants.Atom10Namespace.